### DISCUSSION TITLE
Adding Start Time and End Time makes the Button not show up anymore

### DISCUSSION THREAD

**Discussion by Uncreative-Dev** [add2cal/add-to-calendar-button#502] [Q&A] [answered]
I am using the Wordpress Plugin, but for ease-of-use i have written my own shortcode that builds the Plugin Shortcode.

Following is the HTML i get:

`<add-to-calendar-button timezone="Europe/London" name="Event Name" inline options="'Apple', 'Google', 'iCal', 'Microsoft365', 'MicrosoftTeams', 'Outlook.com', 'Yahoo'" startdate="2023-11-29" enddate="2023-11-29" starttime="17:00" endtime="01:00" atcb-button-id="atcb-btn-1" style="visibility: visible; opacity: 1; position: relative;"></add-to-calendar-button>`

This will not show up. Removing Start and End Time makes the Button work perfectly. Is there anything i am missing?

Thanks in advance for the help.

**Comment by jekuer** [comment, marked as answer]
Hi @Uncreative-Dev ,

You can always add the "debug" option to receive some debugging information in the browser console and also right at where the button is.
In you code, it most probably does not show up because the endTime is before the startTime as startDate and endDate are the same.

Would be great if you could confirm.

All the best,
Jens

**Reply by Uncreative-Dev** [reply]
I didn't even realize that. Changing the dates worked. Thanks for the info with the debug option.
